### PR TITLE
docs(unloading data): add unloading data from databend

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

* add unloading data from databend

Closes #issue

### CODE REVIEW

File: docs/doc/12-unload-data/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Unloading Data from Databend
+slug: ./
+---
+
+Databend provides a variety of tools for loading and unloading data. One of the methods of unloading data from Databend is using the [`COPY INTO <location>`](../14-sql-commands/10-dml/dml-copy-into-location.md) command.
```
Comment (soyeric128) [inline docs/doc/12-unload-data/index.md:6]: Users would want to know what the other ways to unload are if we say "One of the methods of unloading data...". Could you shed some light?

Comment (BohuTANG) [reply docs/doc/12-unload-data/index.md:6]: Like using MySQL client to query all the data into a file, but we don't recommend this way.

File: docs/doc/12-unload-data/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This command is used to export data from a Databend table or query into a file format such as CSV, Parquet, Or JSON.
+
+
+## Unloading Data
+
+### Creating a Stage
+
+Before unloading data, you need to create a stage, which acts as a storage location for the exported data. 
+You can choose to [create an external or internal stage](../14-sql-commands/00-ddl/40-stage/01-ddl-create-stage.md), depending on your requirements.
```
Comment (soyeric128) [inline docs/doc/12-unload-data/index.md:16]: - Is it possible to unload to the user stage, then query from it directly? like "SELECT sum(number) FROM @~ (PATTERN => '.*parquet');"
- I didnt find the document about querying from a stage directly. Can you share the PR of this feature so i can add some document to https://databend-lz99c2p7c-databend.vercel.app/doc/sql-commands/query-syntax/dml-select#from-clause?

Comment (BohuTANG) [reply docs/doc/12-unload-data/index.md:16]: query from the stage is not ready, we are still working on it.

**Comment by soyeric128** [comment]
I made some updates on https://github.com/datafuselabs/databend/pull/10002

**Pull request merged**